src/screens/HomePage/HomePage.js

```diff
@@ -26,7 +26,7 @@ export default function HomePage() {
                 padding: '0 1rem',
               }}
             >
-              <ShareBlock />
+              <ShareBlock centered={true} />
```

Comment:
i think we always want it centered?

Reply:
i feel like it hsould be centered on the location page

Reply:
updated to a wider layout